unittests/inference/classes.py

```diff
@@ -18,8 +18,6 @@ class C:
     x = 1
 
 c = C()
-c.x += True
-d = [1, 2, 3][c.x]
```

Comment:
This was to enforce c.x to have type `int` before adding soft constraints. So we don't need it now.